website/docs/foundations/accessibility/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Accessibility
+caption: Content should be usable and accessible for all users of differing abilities.
+description: Content should be usable and accessible for all users of differing abilities.
 ---
```

Comment:
@heatherlarsen So far I can't think of a good way to start this text off with "Accessibility..." without weakening it. Do you have any ideas or would it be ok to keep as is?

Reply:
Hmm, I can't at the moment. I think it's ok to merge this up, and if we think of something different later, we can iterate on it.

Reply:
Thanks!